.github/workflows/airgap.yml

```diff
@@ -0,0 +1,22 @@
+# Copyright lowRISC contributors (OpenTitan project).
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+name: Airgapped build
+on:
+  pull_request:
+
+jobs:
+  airgapped_build:
+    name: Airgapped build
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: Install system dependencies
+        run: grep '^[^#]' apt-requirements.txt | xargs sudo apt install -y
+      - name: Prepare airgapped environment
+        run: ./util/prep-bazel-airgapped-build.sh
```

Comment:
What's the reason for splitting this to a step?

Reply:
In that case shouldn't the invocation be removed entirely from the ci script?

Reply:
I thought I'd leave it in case people manually run that script and don't want a behaviour change.

Should I just make it implicitly depend on `./util/prep-airgapped-build.sh` instead?

Reply:
I guess it doesn't hurt to keep it..